nose_parameterized/parameterized.py

```diff
@@ -211,11 +211,53 @@ def expand(cls, input, testcase_func_name=None):
             >>>
             """
 
-        def parameterized_expand_wrapper(f):
+        def doc_expansion(f, args, kwargs, is_method):
+            offset = 1 if is_method else 0
+            vns = inspect.getargspec(f)[0][offset:]
+            descs = ["{0} = {1}".format(n, repr(v)) for n, v in zip(vns, args)]
```

Comment:
This is also going to get crashy with mixed str / unicode (I don't think it's unreasonable to assume that there might be unicode in a docstring, and we need to assume that reprs will return some asinine combination of bytes and unicode).

I believe the fix will be:

1. Tweaking `short_repr` to guarantee that it always returns `unicode` (see below), and
2. Using %-formatting or `u` prefixes on the `.format` strings (https://gist.github.com/wolever/be5f554d06d666471283).

```
def short_repr(x, ln=64):
    x_repr = repr(x)
    if isinstance(x_repr, str):
        try:
            x_repr = unicode(x_repr, "utf-8")
        except UnicodeDecodeError:
            x_repr = unicode(x_repr, "latin1")
    if len(x_repr) > n:
        x_repr = x_repr[:n/2] + "..." + x_repr[len(x_repr) - n/2:]
    return x_repr
```